CAPI/API/include/API.h

```diff
@@ -314,16 +315,14 @@ class HumanDebugAPI : public IHumanAPI, public IGameTimer
 
 private:
     std::chrono::system_clock::time_point StartPoint;
+    std::shared_ptr<spdlog::logger> logger;
```

Comment:
是不是 `unique_ptr` 就够了
```suggestion
    std::unique_ptr<spdlog::logger> logger;
```